meta-resin-common/recipes-containers/hostapp-update/files/hostapp-update

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+
+set -eu
+
+local_image=""
+remote_image=""
+reboot=0
+
+while getopts 'f:i:r' flag; do
+	case "${flag}" in
+	f) local_image=$(realpath "${OPTARG}") ;;
+	i) remote_image="${OPTARG}" ;;
+	r) reboot=1 ;;
+	*) error "Unexpected option ${flag}" ;;
+	esac
+done
+
+export DOCKER_HOST="unix:///var/run/docker-host.sock"
+SYSROOT="/mnt/sysroot/inactive"
+
+# Initialize sysroot
+mkdir -p /mnt/sysroot
+for dir in 'dev' 'etc' 'docker' 'hostapps' 'mnt/state' 'proc' 'sbin' 'sys' 'tmp'; do
+	if [ ! -d "$SYSROOT/$dir" ]; then
+		mkdir -p "$SYSROOT/$dir"
```

Comment:
You can use `mkdir -p` and you won't need to check if the directory exists.

Reply:
done